slack_bolt/app/async_app.py

```diff
@@ -188,18 +183,13 @@ def _init_async_middleware_list(self):
         )
         if self._async_oauth_flow is None:
             if self._token:
-                self._async_middleware_list.append(
-                    AsyncSingleTeamAuthorization(
-                        verification_enabled=self._authorization_test_enabled
-                    )
-                )
+                self._async_middleware_list.append(AsyncSingleTeamAuthorization())
```

Comment:
To avoid depending on a specific event loop here, we don't do a blocking operation here (NOTE: `__init__` is not an async function. We cannot have `await` here).